specification/keyvault/data-plane/Microsoft.KeyVault/2016-10-01/keyvault.json

```diff
@@ -2003,7 +2013,8 @@
           "Certificates"
         ],
         "operationId": "GetCertificate",
-        "description": "Gets information about a specified certificate. Authorization: requires the certificates/get permission.",
+        "summary": "Gets information about a certificate.",
+        "description": "Gets information about a specific certificate. It requires the certificates and get permissions.",
```

Comment:
>It requires the certificates and get permissions. [](start = 71, length = 49)

this worded differently from other descriptions referring to required permissions